File: 8_kyu/Expressions_Matter.rs
fn expressions_matter(a: u64, b: u64, c: u64) -> u64 {
    let variations =
    [ a + b + c,
      a + b * c,
      (a + b) * c,
      a * b + c,
      a * (b + c),
      a * b * c ];
    *variations.iter().max().unwrap()
}

File: 8_kyu/Expressions_Matter.jl
"Maximal result for positive a,b,c"
expressionsmatter(a, b, c) = max( maxop(maxop(a,b),c), maxop(a,maxop(b,c)) )

"Maximal result after + or *."
maxop(x,y) = max(x+y,y*x)